app/src/main/java/org/oppia/android/app/player/state/StateFragmentPresenter.kt

```diff
@@ -494,7 +517,7 @@ class StateFragmentPresenter @Inject constructor(
 
   private fun setHintOpenedAndUnRevealed(isHintUnrevealed: Boolean) {
     viewModel.setHintOpenedAndUnRevealedVisibility(isHintUnrevealed)
-    if (isHintUnrevealed) {
+    if (!isHintBulbAnimationEnabled.value) return if (isHintUnrevealed) {
```

Comment:
The spacing here is wrong:

```suggestion
    if (!isHintBulbAnimationEnabled.value) return

    if (isHintUnrevealed) {
```

Reply:
Updated